app/helpers/application_helper.rb

```diff
@@ -371,7 +371,10 @@ def app_metadata
       UISettings: ui_settings, version: SETTINGS[:version].short, docUrl: documentation_url,
       location: Location.current && { id: Location.current.id, title: Location.current.title },
       organization: Organization.current && { id: Organization.current.id, title: Organization.current.title },
-      user: User.current&.attributes&.slice('id', 'login', 'firstname', 'lastname', 'admin')
+      user: User.current&.attributes&.slice('id', 'login', 'firstname', 'lastname', 'admin'),
+      user_settings: {
+        lab_features: Setting[:lab_features],
```

Comment:
Would `def ui_settings` be a better place for this? so all the settings will be in one place?

Reply:
good idea. Moving.